### PR TITLE
Offline schizophrenia

### PR DESCRIPTION
**Pull request by ReoDaProtovali**
## About The Pull Request

A mix of random fixes, changes and additions I made while offline, with no real theme or reason to what I made. Was originally also going to include a system to spawn items in vore bellies when you first spawn in, but that ended up being more work than it was worth.

## Changelog

:cl:
add: Disposal bin autoflush port from TGcode. Whenver you throw a mob into a disposal bin **SUCCESSFULLY**, the bin automatically flushes. This also properly implements throwing people into disposal bins, as it wasnt properly working before this.
add: Added synth emotes to corrupt hounds and subtypes. _*roboboop_
add: Added a maw belly to leopardmanders that autotransfers to their gut. Dunno what made me do that (especially since it ended up being a lot of trouble to get working properly)
add: Added a whole new AI to swoopies based on nurse spiders looking for items to wrap. Right click an AI controlled swoopie and choose the "Change Settings" verb to tell the swoopie to swoop stuff!
add: Added wrappers dropping in vorebellies if you eat and digest the whole food item.
add: Added bones to merc and other humanoid corpses. (leave remains pref is now enabled for them!)
qol: changed default object damage to 0, as it should be.
qol: Made vac-packs not ask if you want to set the destination to borg belly if you're not a borg
image: added the secondary belly layer to the original set of vore bellies. Somehow we missed those and virgo had them. Now we should both have them!
/:cl:

### CODE REVIEW

**Comment by Skysteak** [comment]
I agree with all of these but the Auto-Flush. Getting to disposal people for free because you presumably won the disarm RNG battle is obnoxious, and if you are in a position to get out ,you should be able to get out.

**Comment by Aroliacue** [comment]
Fun little changes, I likey

**Comment by Darlantanis** [comment]
Don't think we need a buff on player-disposals dunking.

**Comment by Raeschen** [comment]
I have a problem with:
> Autoflush. Don't really like the gamery stuff
> 0 thrown damage by default. Throwing something at someone should hurt. It's just 1,

rest seem ok

**Comment by Kazzc** [comment]
`add: Disposal bin autoflush port from TGcode. Whenver you throw a mob into a disposal bin SUCCESSFULLY, the bin automatically flushes. This also properly implements throwing people into disposal bins, as it wasnt properly working before this.`
Very meh, I can see this being abused on mobs/antags/players similar to spam throwing someone against a wall. I'd argue you roleplay your intentions if you want to flush someone down disposals rather than styling on someone with a niche mechanic.

`add: Added bones to merc and other humanoid corpses. (leave remains pref is now enabled for them!)`
Finally, been looking for this change.

`qol: changed default object damage to 0, as it should be.`
I'd argue against this, default object damage is fine. Minor injuries can easily be treated and makes for a more dynamic game if throwing something can do intentional or unintentional harm.

Everything else is pretty good stuff!

**Comment by ReoDaProtovali** [comment]
I've nerfed autoflush to just be autohandle-pull, which should give you a second to escape, if the roll even lands (75% chance, opposed to always getting dunked when thrown)

As for the item damage removal. No. If an item should hurt when thrown, that should be defined. Getting hit by a peice of paper, donut, plush or many other items that make 0 sense dealing damage when thrown should not hurt you. If you want items to deal damage when thrown, define it. It shouldnt be default behavior. Especially since it's 1 force. That's minor enough to not be significant but still enough to cause annoyance with the egregious pain system we have.

**Comment by Raeschen** [comment]
Declined due to unwanted changes

File: code/modules/recycling/disposal.dm [hunk 1]
```diff
@@ -528,14 +528,23 @@
...
 		if(prob(75))
-			if(istype(AM, /obj/item/weapon/holder/micro))
-				log_and_message_admins("[AM] was thrown into \the [src]")
 			AM.forceMove(src)
-			visible_message("\The [AM] lands in \the [src].")
+			if(istype(AM, /obj/item/weapon/holder/micro) || istype(AM, /mob/living))
+				log_and_message_admins("[AM] was thrown into \the [src]")
+				visible_message("\The [AM] lands in \the [src] and triggers the flush system!")
+				//flush() //Away they go! //Uncomment this for proper autoflush. Compromising with autopull to avoid possible disposal dunking abuse
+				flush = 1
```
Comment (Kashargul) [inline code/modules/recycling/disposal.dm:539]: Declined, remove the auto flushing part

File: code/modules/recycling/disposal.dm [hunk 2]
```diff
@@ -734,12 +743,15 @@
 		return
 
 	var/mob/living/U = user
-
+	/* CHOMPEdit: Clong, clong baby.
```
Comment (Kashargul) [inline code/modules/recycling/disposal.dm:746]: Declined section, remove this.

File: code/modules/recycling/sortingmachinery.dm [hunk 3]
```diff
@@ -411,6 +411,22 @@
...
+			if(NORTH)
+				if(AM.loc.y != src.loc.y+1) return ..()
+			if(EAST)
+				if(AM.loc.x != src.loc.x+1) return ..()
+			if(SOUTH)
+				if(AM.loc.y != src.loc.y-1) return ..()
+			if(WEST)
+				if(AM.loc.x != src.loc.x-1) return ..()
+		AM.forceMove(src)
+		src.flush()
```
Comment (Kashargul) [inline code/modules/recycling/sortingmachinery.dm:427]: Declined, remove the auto flushing part

**Review by Kashargul — COMMENTED** [review]
The developer statement had been clear on this. Do not try to push changes against the majority decision of the dev team.

**Pull request closed without merging**